liferay-workspace/tests/blade.portlet.configuration.icon.test/build.gradle

```diff
@@ -0,0 +1,58 @@
+sourceSets {
+	testFunctional {
+		java {
+			compileClasspath += main.output + test.output
+			runtimeClasspath += main.output + test.output
+			srcDir file('src/testFunctional/java')
+		}
+		resources.srcDir file('src/testFunctional/resources')
+	}
+}
+
+configurations {
+	testFunctionalCompile.extendsFrom testCompile
+	testFunctionalRuntime.extendsFrom testRuntime
+}
+
+dependencies {
+	compileOnly "com.liferay.portal:com.liferay.portal.kernel:2.0.0"
+	compileOnly "javax.portlet:portlet-api:2.0"
+	compileOnly "javax.servlet:javax.servlet-api:3.0.1"
+	compileOnly "org.osgi:org.osgi.service.component.annotations:1.3.0"
+
+	testCompile group: "com.liferay.arquillian", name: "com.liferay.arquillian.arquillian-container-liferay", version: "1.0.6"
+	testCompile group: "junit", name: "junit", version: "4.12"
+	testCompile group: "org.jboss.arquillian.graphene", name: "graphene-webdriver", version: "2.1.0.Final"
+	testCompile group: "org.jboss.arquillian.junit", name: "arquillian-junit-container", version: "1.1.11.Final"
+}
+
+startTestableTomcat {
+	checkTimeout = 10 * 60 * 1000
+}
+
+task testFunctional(type: Test) {
```

Comment:
There is a lot of code here to make this test tasks work and they are duplicated for each functional test.

Instead of duplicate this in separate *.test modules, lets put all of these funcitonal tests into seprate Java files but within the same project, so that this gradle config only shows up once.